.github/workflows/monthly_release.yml

```
@@ -2,7 +2,7 @@ name: Monthly release
 
 on:
   schedule:
-    - cron: "0 0 1 * *"  # every first day of the month at midnight
+    - cron: "0 16 2 * *"  # every second day of the month at 08:00 PST (16:00 UTC)
```

Assuming the AMReX release is some time on the 1st, early in the night of the 2nd day of the month might be good
```suggestion
    - cron: "0 9 2 * *"  # every second day of the month at 01:00AM PST (9:00AM UTC)
```